### PR TITLE
Remove QSIPrep-specific tests

### PR DESCRIPTION
**Pull request by tsalo**
Closes none. The one test that combines preprocessing and reconstruction is `dsdti_topup`. Probably need to split that one up into preprocessing- and reconstruction-specific versions.

## Changes proposed in this pull request

- Remove all tests that are preprocessing-specific.

### CODE REVIEW

**Comment by tsalo** [comment]
@mattcieslak do you have ideas on what to do with dsdti_topup?

File: .circleci/config.yml [hunk 1]
```diff
@@ -800,101 +488,13 @@ workflows:
...
-              only: /.*/
-
-      - download_drbuddi_epi:
-          requires:
-            - build
-          filters:
-            tags:
-              only: /.*/
-
       - download_DSDTI:
```
Comment (mattcieslak) [inline .circleci/config.yml:491]: this can go too

File: .circleci/config.yml [hunk 2]
```diff
@@ -800,101 +488,13 @@ workflows:
...
-          requires:
-            - download_DSDTI
-          filters:
-            branches:
-              ignore:
-                - /recon\/.*/
-            tags:
-              only: /.*/
-
       - DSDTI_TOPUP:
```
Comment (mattcieslak) [inline .circleci/config.yml:498]: DSDTI_TOPUP was a test for the prep-then-recon workflow. It isn't possible anymore so it should go

File: .circleci/config.yml [hunk 3]
```diff
@@ -1024,17 +584,7 @@ workflows:
 
       - merge_coverage:
           requires:
-            - DSCSDSI
-            - DSDTI_nofmap
-            - DSDTI_synfmap
             - DSDTI_TOPUP
```
Comment (mattcieslak) [inline .circleci/config.yml:587]:
```suggestion

```

File: .circleci/config.yml [hunk 4]
```diff
@@ -1056,17 +606,7 @@ workflows:
 
       - deployable:
           requires:
-            - DSCSDSI
-            - DSDTI_nofmap
-            - DSDTI_synfmap
             - DSDTI_TOPUP
```
Comment (mattcieslak) [inline .circleci/config.yml:609]:
```suggestion
```

File: qsirecon/tests/utils.py [hunk 5]
```diff
@@ -18,27 +18,14 @@
 def download_test_data(dset, data_dir=None):
     """Download test data."""
     URLS = {
-        "HBCD": "https://upenn.box.com/shared/static/gn1ec8x7mtk1f07l97d0th9idn4qv3yx.xz",
-        "DSCSDSI": "https://upenn.box.com/shared/static/eq6nvnyazi2zlt63uowqd0zhnlh6z4yv.xz",
-        "DSCSDSI_BUDS": "https://upenn.box.com/shared/static/bvhs3sw2swdkdyekpjhnrhvz89x3k87t.xz",
         "DSDTI": "https://upenn.box.com/shared/static/iefjtvfez0c2oug0g1a9ulozqe5il5xy.xz",
```
Comment (mattcieslak) [inline qsirecon/tests/utils.py:21]:
```suggestion
```

File: qsirecon/tests/utils.py [hunk 6]
```diff
@@ -18,27 +18,14 @@
...
             "https://upenn.box.com/shared/static/hr7xnxicbx9iqndv1yl35bhtd61fpalp.xz"
         ),
         "singleshell_output": (
             "https://upenn.box.com/shared/static/9jhf0eo3ml6ojrlxlz6lej09ny12efgg.gz"
         ),
-        "drbuddi_rpe_series": (
-            "https://upenn.box.com/shared/static/j5mxts5wu0em1toafmrlzdndves1jnfv.xz"
-        ),
-        "drbuddi_epi": "https://upenn.box.com/shared/static/plyuee1nbj9v8eck03s38ojji8tkspwr.xz",
         "DSDTI_fmap": "https://upenn.box.com/shared/static/rxr6qbi6ezku9gw3esfpnvqlcxaw7n5n.gz",
```
Comment (mattcieslak) [inline qsirecon/tests/utils.py:28]:
```suggestion
```

**Review by mattcieslak — APPROVED** [review]
Just a couple things that should be deletable.

**Pull request merged**